Shared/Sources/NewDataSource.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import UIKit
+
+/// The purpose of this class is to eventually replace the current custom data source class (`DataSource`).
+/// The existing custom class is more complex and incompatible with models that extend `Decodable`.
+final class NewDataSource<Model: Decodable, Cell: UITableViewCell>: NSObject, UITableViewDataSource {
+  typealias Models = [[Model]]
```

Comment:
Is that… is that an array of arrays? 🤢

Reply:
Yes 😭!
So this class was written before I created a custom protocol `SectionItem` to avoid nested Array and I forgot to replace it. Decheng has done a great job to point it out!